solr/CHANGES.txt

```diff
@@ -236,6 +236,8 @@ Bug Fixes
 
 * SOLR-16656: rid parameter missing from query logs (Alex Deparvu)
 
+* SOLR-16676: Http2SolrClient loss of MDC context (Alex Deparvu)
```

Comment:
I like succinctness but here it's unclear.

```suggestion
* SOLR-16676: Logs: Http2SolrClient.async() lacked MDC (Alex Deparvu)
```

If this helps distributed search (which uses this I think) then mention that.  But I haven't noticed such a problem so maybe it doesn't?